packages/laboratory/src/sequelize_laboratory/models/benchmark.ts

```diff
@@ -18,7 +18,7 @@ export class Benchmark extends Model<Benchmark> implements IBenchmark {
   @Column(DataType.STRING)
   apiVersion!: string;
 
-  // TODO: REVIEW: magic number 1024
-  @Column(jsonColumn<PipelineStage[]>('stages', 1024))
+  // TODO: REVIEW: magic number 4096
```

Comment:
think you already know the comment from the todo

Reply:
This & its peer TODOs have been here since forever. Will clean this up